kuma/javascript/src/l10n.js

```diff
@@ -136,3 +137,32 @@ export function interpolate(s: string, args: Array<any> | { [string]: any }) {
         );
     }
 }
+
+const ELEMENT_REGEXP = /(<\w+\s*\/>)/g;
+/**
+ * Similar to interpolate() it interpolates the given id-string with the given
+ * properties, but it can also work with React Elements. It will replace strings
+ * of the form `<name/>` and return an Array of React Elements.
```

Comment:
Could we add an example here in the comment of a simple use case? Until I saw it in action in the newsletter component I was not 100% sure of how to use it.